CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.2.3
+
+- Fix #90: Apply locale based messaging
+- Issue #65 Adding cucumber tests for proxy settings
+- Issue #65 Adding proxy options to rhn_register and subscription_manager
+- Issue #65 Introducuing minitest as unit test framework and setting up basic test harness for Vagrant plugins
```

Comment:
@budhrg It seems `Issue #65 Introducing minitest as unit test framework and setting up basic test harness for Vagrant plugins` it could have been a separate issue. I will just remove the issue number from the statement.